docs/source/keywords.rst

```diff
@@ -575,10 +575,15 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
       - Models
       - Notes
 
+    * * ``Kokkos_ARCH_AMD_GFX942_APU``
+      * GFX942
+      * MI300A
+      * (since Kokkos 4.5)
+
     * * ``Kokkos_ARCH_AMD_GFX942``
       * GFX942
       * MI300A, MI300X
-      * (since Kokkos 4.2)
+      * (since Kokkos 4.2, since Kokkos 4.5 this should only be used for MI300X)
```

Comment:
I haven't looked at how this renders but I expects this folds over multiple lines.
Did it still look fine to you, should we instead do a "see below"?

Reply:
Looks ok to me:
![image](https://github.com/user-attachments/assets/03627c32-4626-4f75-a68a-11ba7e5bce96)

Reply:
Sure